emulator/blockchain.go

```diff
@@ -1699,3 +1741,21 @@ func (b *Blockchain) GetSourceFile(location common.Location) string {
 	return location.ID()
 
 }
+
+func (b *Blockchain) systemChunkTransaction() (*flowgo.TransactionBody, error) {
+	script := templates.ReplaceAddresses(
+		systemChunkTransactionTemplate,
+		templates.Environment{
+			RandomBeaconHistoryAddress: b.GetChain().ServiceAddress().Hex(),
+		},
+	)
+
+	tx := flowgo.NewTransactionBody().
+		SetScript([]byte(script)).
+		SetGasLimit(9999).
```

Comment:
use the flow-go constant for max gas

Reply:
Good point :+1: 
Updated in https://github.com/onflow/flow-emulator/pull/511/commits/e1686844a60e3165344b08bb9be89fa695a5bf26